docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -1314,6 +1314,62 @@ See the proofs for this approach below.
+  t_end     := time when simulation ends, in years since start (e.g. 6 years)
+  20_weeks  := 20 weeks, measurd in years (e.g. 20*7/365.25)
+
+  # Population level parameters
+  hb_shift              = effect size on hemoglobin (normally disributed, mean 3.0 g/L)
+  bw_response           = dose-response on birthweight (normally distributed, mean 15.1/10 g/mg)
+  baseline_coverage(t)  = iron fortification coverage at time t in baseline scenario
+  delta_coverage(t)     = change in coverage from baseline to intervention
+  coverage(t)           = iron fortification coverage at time t
+                        = baseline_coverage(t) + delta_coverage(t)
+    # Multiplier on hb_shift due to age
+  hb_age_fraction(age)  = 0 if age<0.5 else (age-0.5)/1.5 if age<2 else 1
+    # Multiplier on hb_shift due to lag in response time (=1 if simulant hasn't yet received fortification)
+  hb_lag_fraction(time_since_coverage)
+                        = time_since_coverage/0.5 if 0<=time_since_coverage<0.5 else 1
+  iron_concentration    = concentration of iron in the country's fortified flour
+
+  # Individual level attributes
+  p_i = propensity of simulant and mother for exposure to unfortified flour
+  daily_flour_i = average daily flour consumption of simulant's mother
+  gestational_age_i = gestational age of simulant drawn from GBD (in years)
+  birthweight_gbd_i = birthweight drawn from GBD for simulant
+  age_i(t) = the simulant's age at time t
+  hb_gbd(age_i(t)) = hemoglobin level drawn from GBD for simulant
+  birthweight_i := simulant's birthweight after adjusting for mother's iron consumption
+  hb_i(t) := simulant's hemoglobin level after adjusting for fortification
+
+  # Stratification on child coverage - assign categories on simulant initialization
+  child_fortified_baseline_end_i = p_i < baseline_coverage(t_end)
+  child_fortified_end_i = p_i < coverage(t_end)
+
+  # At beginning of simulation
+  mother_fortified_i = "unknown"
+  mother_fortified_baseline_i = "unknown" # For stratification
+  birth_weight_i = birthweight_gbd_i
+  child_fortified_i(0) = p_i < coverage(0)
+  hb_i(0) = hb_gbd(age_i(0)) + (-coverage(0)+child_fortified_i(0)) * hb_shift *  hb_age_fraction(age_i(0))
+
+  # When simulant is born in the simulation
+  mother_fortified_i = p_i < coverage(t - gestational_age_i + 20_weeks)
+  mother_fortified_baseline_i = p_i < baseline_coverage(t - gestational_age_i + 20_weeks) # For stratification
+  bw_shift_i = bw_response * iron_concentration * daily_flour_i
+  birth_weight_i = birthweight_gbd_i + (-coverage(t)+mother_fortified_i) * bw_shift_i
+
+  # At each simulation time step (including when simulant is born)
+  child_fortified_i(t) = p_i < coverage(t)
+  time_covered_i = argmin_t(child_fortified_i(t) == True) # =float('inf') if simulant never gets covered
+  hb_i(t) = hb_gbd(age_i(t)) + (-coverage(t)+child_fortified_i(t)) * hb_shift *  hb_age_fraction(age_i(t)) * hb_lag_fraction(t-time_covered_i)
```

Comment:
@aflaxman @alibow I think this equation has an error. I think instead it should be:

`hb_i(t) = hb_gbd(age_i(t)) + (-coverage(t)+child_fortified_i(t) * hb_lag_fraction(t-time_covered_i)) * hb_shift *  hb_age_fraction(age_i(t))`

Basically, this changes `child_fortified_i(t)` from a step function, which jumps immediately from 0 to 1 at `t=time_covered_i`, to a continuous function that ramps up from 0 to 1 over 6 months, starting at `t=time_covered_i`.

I think the current version jumps from `hb_gbd(age_i(t)) - coverage(t) * hb_shift *  hb_age_fraction(age_i(t))` to `hb_gbd(age_i(t))` at time `t=time_covered_i`, then ramps up to the full effect size over 6 months from that point.  That doesn't seem like what we want. This piece is supposed to implement the hemoglobin effect described in the section `Determining Whether A Simulant is Affected - Iron`.

I'm going to go ahead and make this change before merging to master. Since you already approved the PR, let me know if you think the new version looks correct.

Reply:
Hmm, actually I think the issue here is that we are not differentiating between baseline and intervention coverage.

My thought is that baseline coverage should be defined as:
hb_baseline_i(t) = hb_gbd(age_i(t)) + (-coverage(t)+child_fortified_i(t)) * hb_shift *  hb_age_fraction(age_i(t)) 
^ No need to include the hb_lag_fraction here because we are assuming that time since coverage is sufficient for everyone.

And intervention coverage should be defined as:
hb_intervention_i(t) = hb_baseline_i(t) + new_coverage_i * hb_shift * hb_age_fraction(age_i(t)) * hb_lag_fraction(t-time_covered_i)

Where new_coverage_i is an indicator variable for delta coverage (would need to be a function of time).

Reply:
Should your baseline coverage equation be:
`hb_baseline_i(t) = hb_gbd(age_i(t)) + (-baseline_coverage(t)+child_fortified_baseline_i(t)) * hb_shift * hb_age_fraction(age_i(t))` ?

If so, I think the new version I wrote down is equivalent to this and works in both the baseline and intervention scenarios, because if the child is covered in baseline, then `time_covered_i = float('-inf')` and hence `hb_lag_fraction(t-time_covered_i) = `hb_lag_fraction(float('inf')) = 1`.  But perhaps it would be clearer to separate out the baseline level explicitly as you suggest...

Reply:
Oh, ok! I didn't understand that at first pass. And yes, about child_fortified_baseline_i(t) in my equation.

If those equations are equivalent then I think it looks good!

Reply:
Actually, my version was wrong because I used `coverage(t)` instead of `baseline_coverage(t)`. This is corrected in the latest commit.